### ISSUE TITLE
Adding a user to ignore list gives no feedback.

### ISSUE THREAD

**Issue by Daenyth** [Cockatrice/Cockatrice#1875]
When you right click on a user in the user list, and add them to ignore list there's no indication that you successfully ignored them.

It should pop up a desktop notification

**Comment by radioactivehazards** [comment]
the silence is the indication no?

**Comment by Jinxstands4Jinx** [comment]
Would it be possible for this to inform the ignored user as well?

**Comment by Daenyth** [comment]
No, it's a design decision that users don't know when other people have them ignored

**Comment by Jinxstands4Jinx** [comment]
seems odd. Maybe it's better this way but I personally like to know if people ignore me, so I can in return ignore them. It saves on times when you may be talking to ghosts. It also stops the 3rd party convos about people trying to get off other users ignore lists. And would help "accidental" ignores which I have had happen before. Just ideas. If its meant to be like that than ok no harm. Just offered idea.

**Comment by Daenyth** [comment]
> It also stops the 3rd party convos about people trying to get off other users ignore lists

Generally a moderator should be doing that.

When ignores get notified, it tends to result in the person being ignored going further out of their way to bypass it. It doesn't help the person doing the ignoring at all.

**Comment by Jinxstands4Jinx** [comment]
Actually, I did not think of that. I did not see that being told you are ignored could make a person mad enough to go through hoops to bypass the ignoring...I guess some people may have the time. My bad.

**Comment by tooomm** [comment]
So the issue is about showing a confirmation to yourself, not the other (ignored) user.

**Comment by Daenyth** [comment]
> So the issue is about showing a confirmation to yourself, not the other (ignored) user.

Exactly correct.

**Comment by ZeldaZach** [comment]
I'd be fine with a popup saying "User X added to your ignore list"
